WowUp.WPF/Utilities/StartupOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommandLine;
```

Comment:
Lets move this class to Models.WowUp instead of Utilities.

Reply:
Sure